lua/octo/pickers/telescope/provider.lua

```diff
@@ -1204,28 +1204,46 @@ local function mark_notification_read()
   end
 end
 
-function M.notifications()
+function M.notifications(opts)
+  opts = opts or {}
   local cfg = octo_config.values
-  local opts = {
-    preview_title = "",
-    prompt_title = "Github Notifications",
-    results_title = "",
-  }
+
+  if cfg.notifications.current_repo then
+    opts.repo = utils.get_remote_name()
+  end
+
+  local endpoint = "/notifications"
+  opts.prompt_title = "Github Notifications"
+  if opts.repo then
+    local owner, name = utils.split_repo(opts.repo)
+    endpoint = string.format("/repos/%s/%s/notifications", owner, name)
+    opts.prompt_title = string.format("%s Notifications", opts.repo)
+  end
```

Comment:
```suggestion
  local endpoint = "/notifications"
  if opts.repo then
    local owner, name = utils.split_repo(opts.repo)
    endpoint = string.format("/repos/%s/%s/notifications", owner, name)
  end
  opts.prompt_title = opts.repo and string.format("%s Notifications", opts.repo) or "Github Notifications"
```

i think a short circuit is slightly more readable then overwriting the variable